### DISCUSSION TITLE
supabase client python returns html on query?

### DISCUSSION THREAD

**Discussion by dragonballa** [supabase/supabase#22459] [Questions]
self.supabase = create_client(supabase_url, supabase_key) #i also tried supabase: Client = create_client()
        print(self.supabase.table("profiles").select("*").execute())

postgrest.exceptions.APIError: {'message': 'JSON could not be generated', 'code': 500, 'hint': 'Refer to full message for details', 'details': 'b\'<!DOCTYPE html>\\n\\n\\n\\n <html class="no-js" lang="en-US"> \\n<head>\\n<title>Worker threw exception

but the rest API via curl works fine....

**Comment by k00sharath** [comment]
Were you able to resolve this issue
